docs/setup_docs.sh

```diff
@@ -41,10 +41,10 @@ cd tmp
 
 # Since there's no documentation yet available for a release branch,
 # we only get the documentation from the main branch
-integrate_connector_docs elasticsearch v3.0.0-docs
-integrate_connector_docs aws v4.1.0-docs
-integrate_connector_docs opensearch v1.0.0
-integrate_connector_docs mongodb v1.0.0-docs
+integrate_connector_docs elasticsearch v3.0
```

Comment:
I see that you have created a similar PR: #22402
Why not fix this in one PR?

Reply:
Thanks @healchow for the review.
 
> I see that you have created a similar PR: https://github.com/apache/flink/pull/22402 
Why not fix this in one PR?

In fact, these two PRs were created to pick the changes in A into the release-1.17 and release-1.16 branches([BP-xx] can also be seen from the title). Otherwise, how to merge code into two different branches in one PR?

Reply:
For some changes that can be determined not to cause problems due to pick, the committer does not need to create a specific backport pr. But generally, we create it to check if CI can pass.